modules/game_interface/gameinterface.lua

```diff
@@ -362,6 +362,30 @@ function updateStretchShrink()
   end
 end
 
+function addToPanels(uiWidget)
+  uiWidget.onRemoveFromContainer = function(widget)
+    if gameLeftPanel:isOn() then
+      if widget:getParent():getId() == 'gameRightPanel' then
+        if gameLeftPanel:getEmptySpaceHeight() - widget:getHeight() >= 0 then
+          widget:setParent(gameLeftPanel)
+        end
+      elseif widget:getParent():getId() == 'gameLeftPanel' then
+        if gameRightPanel:getEmptySpaceHeight() - widget:getHeight() >= 0 then
+          widget:setParent(gameRightPanel)
+        end
+      end
+    end
+  end
+
+  if not gameLeftPanel:isOn() then uiWidget:setParent(gameRightPanel) return end
```

Comment:
I don't like to use one-liners
Otherwise LGTM

Reply:
I changed one-line function to multi-line.